src/entity/User/PetFactory.java

```diff
@@ -0,0 +1,7 @@
+package entity.User;
+
+import entity.Pet;
+
```

Comment:
i think this should be a parameterized constructor that initializes the fields of pet thru the arguments, similar to how adopterUserFactory. refer to those